e2e/LongHaul/device/IotHub.cs

```diff
@@ -66,8 +67,13 @@ public IotHub(Logger logger, Parameters parameters)
         /// <summary>
         /// Initializes the connection to IoT Hub.
         /// </summary>
-        public async Task InitializeAsync()
+        public async Task InitializeAsync(CancellationToken? ct = null)
         {
+            if (ct != null)
```

Comment:
Share it when the program inits so we can reference it from the connection status callback.